src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -93,8 +96,9 @@ def __init__(
         self.do_normalize = do_normalize
         self.image_mean = image_mean if image_mean is not None else [0.485, 0.456, 0.406]  # ImageNet mean
         self.image_std = image_std if image_std is not None else [0.229, 0.224, 0.225]  # ImageNet std
+        self.num_labels = num_labels
 
-    def _resize(self, image, size, target=None, max_size=None):
+    def _resize_with_size_divisibility(self, image, size, target=None, max_size=None, **kwargs):
```

Comment:
Can you avoid the `**kwargs`, I'm not using it in any feature extractor and it's unclear what this method accepts in that case

Reply:
even in a non-user-facing private method?

Reply:
If you can avoid it, yes.

Reply:
Also I just checked, the `kwargs` is not useful here.

Reply:
Btw, I typically also make the `resample` an attribute that can be configured.

Reply:
I am not sure I understood, no user should call this method since it is marked as `_` private